src/js/components/projects/detail.tsx

```diff
@@ -130,6 +184,35 @@ const ProjectDetail = (props: RouteComponentProps) => {
           { name: project.name },
         ]}
         onRenderHeaderActions={onRenderHeaderActions}
+        sidebar={[
+          <div
+            key="addmember-section"
+            className="slds-m-bottom_medium add-member"
+          >
+            <h2 className="slds-text-heading_medium slds-p-bottom_small">
+              {i18n.t('Collaborators')}
```

Comment:
I wasn't sure if we should use the word `members` here or `collaborators`. Anyone have an opinion?

Reply:
I like collaborators